start.go

```diff
@@ -73,6 +74,14 @@ func NewGlobal(conf *Config, sink MetricSink) (*Metrics, error) {
 	return metrics, err
 }
 
+// GetConfig returns the Config for the globalMetrics instance if available
+func GetConfig() (*Config, error) {
```

Comment:
I think it should just return *Config which may be nil, no need for an error return.